dotnet/src/Connectors/Connectors.AI.OpenAI/ChatCompletion/OpenAIChatCompletion.cs

```diff
@@ -59,4 +60,9 @@ public ChatHistory CreateNewChat(string instructions = "")
     {
         return this.InternalCompleteTextUsingChatAsync(text, requestSettings, cancellationToken);
     }
+
+    public IAsyncEnumerable<string> CompleteStreamAsync(string text, CompleteRequestSettings requestSettings, CancellationToken cancellationToken = default)
+    {
+        throw new System.NotImplementedException();
```

Comment:
Instead of throwing here, if it's not supported, I'd just call the normal CompleteAsync and then return as a valid (but completed) IAsyncEnum<> with the whole string result. (Same for each of the other client types)

Out of curiosity, what happens if you call this method on the OpenAI SDK, just like the Azure one but using OAI endpoint? Does it throw?

Reply:
That makes sense, did the change with the fallback implementation for Hugging Face and Chat Clients.